dev_guide/builds.adoc

```diff
@@ -5,19 +5,40 @@
 :icons:
 :experimental:
 :toc: macro
+:toclevels: 3
 :toc-title:
 :prewrap!:
 
 toc::[]
 
 == Overview
-A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build] is a process of creating
-runnable images to be used on OpenShift. There are three build strategies:
+
+A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build]
+is a process of creating runnable images to be used on OpenShift.
+
+Build configurations are characterized by strategy and sources. The strategy
```

Comment:
Suggest (but not strongly, see below): "... by a *strategy* and one or more *sources*."

@vikram-redhat 
We should probably add to the guidelines a blurb on how to refer to terms in a definition context.